### ISSUE TITLE
Module can only be default-imported using the 'esModuleInterop' flag

### ISSUE THREAD

**Issue by BlackAsLight** [PrismarineJS/prismarine-registry#8]
- [x] The [readme](https://github.com/PrismarineJS/prismarine-template/README.md) doesn't contain a resolution to my issue 
- [x] The [example](https://github.com/PrismarineJS/prismarine-template/example.js) doesn't contain a resolution to my issue 

## Versions
 - node: 17.4.0
 - prismarine-template: ?

## Detailed description of a problem
When trying to compile my code, `tsc main.ts` it gives me this error
```
node_modules/prismarine-registry/index.d.ts:1:8 - error TS1259: Module '"/Users/soul/Projects/MC Clicker/node_modules/minecraft-data/index"' can only be default-imported using the 'esModuleInterop' flag

1 import mcData from 'minecraft-data'
         ~~~~~~

  node_modules/minecraft-data/index.d.ts:938:1
    938 export = MinecraftData
        ~~~~~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

Found 1 error.
```

```js
require('dotenv').config()
import * as mineflayer from 'mineflayer'
import { Movements, pathfinder } from 'mineflayer-pathfinder'
import { mineflayer as mineflayerViewer } from 'prismarine-viewer'
import mcData = require('minecraft-data')

const bot = mineflayer.createBot({
	host: process.env.HOST || 'localhost',
	username: process.env.USERNAME || 'PlayerOne',
	password: process.env.PASSWORD,
	auth: (() => {
		const auth = process.env.AUTH
		if (auth && (auth === 'mojang' || auth === 'microsoft'))
			return auth
		console.warn('AUTH in .env did not equal `mojang` or `microsoft`, defaulting to `mojang`.')
		return 'mojang'
	})(),
	port: parseInt(process.env.PORT || '0') || 25565
})
bot.loadPlugins([pathfinder])

bot.once('spawn', async () => {
	mineflayerViewer(bot, {
		port: 3000,
		firstPerson: true
	})
	bot.pathfinder.setMovements((() => {
		const movements = new Movements(bot, mcData(bot.version))
		movements.scafoldingBlocks = []
		movements.canDig = false
		return movements
	})())
})
```

## Expected behavior
It compiling normally. 

## Additional context
From my own tests it doesn't like me doing `import mcData from 'minecraft-data'` either, but doing `import mcData = require('minecraft-data')` seems to be okay.

I've look online for a solution and the internet said to set `esModuleInterop` true in my tsconfig.json, but it already is.

---

**Comment by extremeheat** [comment]
This is a problem in node-mc-data, https://github.com/PrismarineJS/node-minecraft-data/blob/41f3a475078fee8704822c9d8f82bce7f70f9442/typings/generate-typings.js#L32 should be doing a default export

**Comment by BlackAsLight** [comment]
Should I open up this issue there then?

**Comment by extremeheat** [comment]
You can open a PR to fix it there, yeah. I don't believe it's an issue here.

**Comment by sertonix** [comment]
This is an problem here. This type definition is not compatible with `esModuleInterop: false`. I fixed it in #13
